### DISCUSSION TITLE
Associate space item with yabai label

### DISCUSSION THREAD

**Discussion by basbebe** [FelixKratz/SketchyBar#155] [Q&A] [answered]
If I'm not mistaken, yabai labels move with the space.
So if I move a space to another position or to another screen, the  label moves with it.

Has anyone already written a script to have items in SketchyBar associated with a yabai label instead of a screen number?

Could imagine that https://github.com/FelixKratz/SketchyBar/discussions/128#discussioncomment-1806189 already goes in this direction?

I would like my space items order to change when I move a space / move it to another screen and show the correct items when I moved a screen.

**Comment by FelixKratz** [comment, marked as answer]
This is a good starting point I think, since yabai has no space move event a custom event is needed that is triggered via eg skhd : https://github.com/FelixKratz/SketchyBar/discussions/12#discussioncomment-1670992

**Reply by basbebe** [reply]
WIP solution posted there: https://github.com/FelixKratz/SketchyBar/discussions/12#discussioncomment-1983405
